### PR TITLE
Fix mongo tests

### PR DESCRIPTION
**Pull request by mmanciop**

### CODE REVIEW

File: test/instrumentations/express/app/express_app.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ app.get('/basic', async (_, res) => {
 
 server = app.listen(0, () => {
   const port = server.address().port;
-  console.info('Listening on port ' + port);
+  console.error('Listening on port ' + port);
```
Comment (therightstuff) [inline test/instrumentations/express/app/express_app.js:50]: hmmm... i was under the impression that if you console.log, then the logs will always be displayed but on stdout, which seems like the best thing to do in this case

File: test/instrumentations/mongodb/mongodb.test.ts [hunk 2]
```diff
@@ -31,27 +31,56 @@ const expectedIndexStatement = expect.stringMatching(
 );
 
 describe.each(versionsToTest('mongodb', 'mongodb'))(
-  "Instrumentation tests for the 'mongodb' package",
-  function (versionToTest) {
-    let testApp: ChildProcessWithoutNullStreams;
+  'Instrumentation tests for the mongodb package',
+  (versionToTest) => {
```
Comment (therightstuff) [inline test/instrumentations/mongodb/mongodb.test.ts:35]: there's a reason that i switched away from all the arrow functions - in both jest and mocha, arrow functions break things. unless there's some important reason for arrow functions here that i'm not familiar with, please return these

File: test/utils/test-apps.ts [hunk 3]
```diff
@@ -5,113 +5,130 @@ import { Span, readSpanDump } from './spans';
...
-            ...process.env, ...{
-                OTEL_SERVICE_NAME: serviceName,
-                LUMIGO_DEBUG_SPANDUMP: spanDumpPath,
-                LUMIGO_DEBUG: String(true),
-                ...env_vars
-            }
-        },
-        shell: true,
-    });
+export class TestApp {
```
Comment (therightstuff) [inline test/utils/test-apps.ts:10]: excellent improvement 👍 👍

**Review by therightstuff — CHANGES_REQUESTED** [review]
overall, 👍 👍 
i have two concerns, though: the first, the use of arrow functions in the tests, and the second, that `test:untested-instrumentations` is no longer working locally and the [push-branch job](https://github.com/lumigo-io/opentelemetry-js-distro/actions/runs/5490322947/jobs/10005827746?pr=189) is broken.

**Comment by mmanciop** [comment]
I don't understand the concern about arrow functions in tests. The other, yes, that prevents the PR from merging. @therightstuff can you please give it a try?

File: test/instrumentations/mongodb/mongodb.test.ts (hunk 2 above)
Comment (mmanciop) [reply test/instrumentations/mongodb/mongodb.test.ts]: Done in 367dcee

File: test/instrumentations/express/app/express_app.js (hunk 1 above)
Comment (mmanciop) [reply test/instrumentations/express/app/express_app.js:50]: In the test app we explicitly listen to `stderr`

File: scripts/test_untested_versions.js [hunk 4]
```diff
@@ -16,7 +16,13 @@ const instrumentationsFolders = fs.readdirSync('src/instrumentations').filter(fu
   return isDirectory && hasTestedVersionsFile;
 });
 
-for (const package of instrumentationsFolders) {
+let instrumentationToTest = instrumentationsFolders.filter(instrumentation => {
```
[the remark this answers is not in the document]
Comment (mmanciop) [reply scripts/test_untested_versions.js:19]: README is for users, not contributors

[the remark this answers is not in the document]
Comment (mmanciop) [reply scripts/test_untested_versions.js:19]: Done in 6dbf9d1dd2ac87fe107ba119fe265af39a3f2789

**Review by therightstuff — APPROVED** [review]

**Pull request merged**